src/components/SearchForm/SearchForm.spec.tsx

```diff
@@ -0,0 +1,77 @@
+describe('SearchForm', () => {
```

Comment:
The tests are failing. 
<img width="1616" alt="Screen Shot 2022-01-19 at 10 27 27 AM" src="https://user-images.githubusercontent.com/11657599/150161833-384ae4a2-2b22-4811-ab85-bff63a969515.png">

Reply:
fixed, was a typo in url